### DISCUSSION TITLE
Is it possible to filter like "LIKE"?

### DISCUSSION THREAD

**Discussion by alexander161198** [ChilliCream/hotchocolate#4458] [Q&A]
Hello!

Is there a way to filter like the "LIKE" SQL operator? 
I want to do something like that:

```
query GetHotels
{
  hotels(where: { name: { like: "New%"} } )
  {
    nodes
    {
      id,
      name,
      rating
    }
  }
}
```

Or maybe even use regular expressions.

**Comment by michaelstaib** [comment]
We support contains

**Reply by alexander161198** [reply]
Is there any way to filter strings, for example, like that: "[A-Z]%text"

**Reply by kolpav** [reply]
@alexander161198 Take a look here https://chillicream.com/docs/hotchocolate/api-reference/extending-filtering

**Reply by alexander161198** [reply]
Ok. Thanks
